app/helpers/comments_helper.rb

```diff
@@ -40,6 +54,16 @@ def render_pings(comment, pingable: nil)
     end.html_safe
   end
 
+  # Converts all ping strings (i.e. @#1234) in content into usernames for use in text-only contexts
+  # @param content [String] content to convert ping strings for
+  # @return [String] processed content
+  def render_pings_text(content)
+    content.gsub(/@#\d+/) do |id|
+      user = User.where(id: id[2..-1].to_i).first
```

Comment:
What does this do if a comment pings more than one user?  (I'm wondering about the `first` call here.)

Reply:
`id[2..-1]` takes a slice from the ping string, which is `@#<numeric id>`, resulting in just a numeric id. Then, `User.where(id: <resulting numeric id>).first` ensures only one user is returned. This is then repeated for each ping string due to how `gsub` works.

Tbh, this is not very efficient, but improving upon the logic is out of scope for this PR - this is basically an exact copy of what's in `render_pings` above

Reply:
Thanks for explaining, and I agree it's fine to do it this way here.